tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -26,12 +26,12 @@
 #include "ROOT/RStringView.hxx"
 #include "ROOT/RVec.hxx"
 #include "ROOT/TBufferMerger.hxx" // for SnapshotHelper
-#include "ROOT/RCutFlowReport.hxx"
-#include "ROOT/RDFUtils.hxx"
+#include "ROOT/RDF/RCutFlowReport.hxx"
+#include "ROOT/RDF/Utils.hxx"
```

Comment:
I actually used RDFUtils in my own datasource. Is this header really private or simply moved to a better place? I think retaining the ability to write datasources in user code is very important to us.

Reply:
Simply moved to a better place, you can keep including it (with the new name).

"private" is too strong of a word, but the entities it contains are in namespace `ROOT::Internal`, so we might modify them (or move them, like in this case) without notice.

What are you using from this header exactly? Maybe we should promote it to `ROOT::Detail` (expert-only but not "private").

Reply:
Not sure, actually. I will play a bit...

Reply:
In general, having external projects rely on entities in `ROOT::Internal` is bad. Let us know what classifies as generally useful and we will promote it to `ROOT::Detail`.